src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  position: relative;
+}
+
+.icon {
+  height: 40px;
+  width: 40px;
+  position: absolute;
+  left: 50px;
+  top: 10px;
+  bottom: 10px;
+}
+
+.nav__list {
+  font-family: "Roboto", serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  /* color: #000; */
+}
+
+.nav__link {
+  text-decoration: none;
+  line-height: 60px;
+  color: #000;
+}
+
+/* .nav__link:visited {
+  color: #00acdc;
+} */
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__item {
+  display: inline;
+  padding-top: 24px;
+  padding-right: 17.02px;
+  cursor: pointer;
+}
```

```suggestion
.nav__item:not(:last-child) {
   margin-right: 20px;
}
```
this styles wil be redundant for .nav__item
item of list will be responsible only horizontal margin between other items
see comment about place .nav__link inside .nav__item